package.json

```diff
@@ -23,7 +23,6 @@
     ]
   },
   "devDependencies": {
-    "@elastic/eslint-import-resolver-kibana": "link:../../packages/osd-eslint-import-resolver-opensearch-dashboards",
```

Comment:
I would modify: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/.eslintrc.yml#L6 as well.

From my understanding this dependency has it's use cases but I don't see it being used within this plugin repo that makes it worth linking.

If this was a shared repo in the past, or if you need to extend the imports rules then I get see it. But after removing and modifying the eslintrc, I needed to modify the import orders as I believe our lint rules kicked in for import order and it didn't treat your plugin as the root directory